app/src/main/java/io/neurolab/tools/BaselineCorrectionFunction.java

```diff
@@ -0,0 +1,33 @@
+package io.neurolab.tools;
+
+import android.annotation.TargetApi;
+
+import java.util.function.Function;
+
+import io.neurolab.MathBasics;
+
+@TargetApi(24)
```

Comment:
This makes this complete class unfunctional and not usable below API 24. Please change.